Month2/Pointers_Exercise/Section1Pointers/Exercise6/Exercise6.cpp

```diff
@@ -0,0 +1,28 @@
+#include<iostream>
+#include <stdlib.h>
+#include <time.h>
+ 
+int main()
+{
+	int Array[10] ;
+	int* Ptr = Array;
+	srand(time(0));
+
+	
+	std::cout << "I have looped through the following numbers:" << std::endl;
+	for (Ptr = Array; Ptr < Array; Ptr++) 
+	{
+		*Ptr = rand();
+		std::cout << *Ptr<< "\t";
```

Comment:
Same here, did you test this? Doesn't seem like this should work as intended. You're assigning Ptr to Array and then looping while Ptr is less than Array, which should never be the case.